views.js

```diff
@@ -245,16 +245,21 @@ function renderMessages (state, width, height) {
 
   // Character-wrap to area edge
   var allLines = msgs.reduce(function (accum, msg) {
-    const nickLength = msg.raw.author ? msg.raw.author.length : 0
-    var indent = 0
-    if (state.config.messageIndent === 'time' ||
-        state.config.messageIndent === 'nick') {
-      indent += state.messageTimeLength + 1 // + space
+    // Status message
+    if (!msg.timestamp) {
+      // TODO(kira): These don't wrap yet & ought to!
+      accum.push(' * ' + msg.formatted)
+      return accum
     }
-    if (state.config.messageIndent === 'nick') {
-      indent += nickLength + 3 // + space and <>
-    }
-    accum.push.apply(accum, util.wrapAnsi(msg.formatted, width, indent))
+
+    const indent = util.strwidth(msg.timestamp + msg.author + msg.emote) + 1
```

Comment:
i didn't follow what the `+1` was, could you add a comment that explains what it is?

Reply:
It's the space between the `09:13pm <foo>` author prefix and the actual message content. I've revised this a bit so there's no `+ 1`.